agent/agentsocket/socket_unix.go

```diff
@@ -1,9 +1,6 @@
-//go:build !windows
-
```

Comment:
This change has me a bit confused. Why did we initially add not-implemented methods on Windows, and then remove it? Behavior when trying to use this on Windows is now somewhat undefined AFAICT.